site/src/components/WorkspaceActions/DropdownContent/DropdownContent.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import { Story } from "@storybook/react"
+import { DeleteButton, StartButton, StopButton } from "../ActionCtas"
+import {
+  ButtonMapping,
+  ButtonTypesEnum,
+  WorkspaceStateActions,
+  WorkspaceStateEnum,
+} from "../constants"
+import { DropdownContent, DropdownContentProps } from "./DropdownContent"
+
+export default {
+  title: "components/DropdownContent",
```

Comment:
I would add more context to this name in the Storybook, maybe `WorkspaceActionsDropdown`.